components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/ApplicationListItem.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Let's fix the other places as well.

```suggestion
 * Copyright (c) 2023-2024, WSO2 LLC. (http://www.wso2.com).
```

Reply:
This class is generated through the API generator, which handles adding headings and other updates. Typically, we do not modify these generated contents.

Reply:
Ack, my bad i didn't see it was inside gen.

Reply:
Going forward as our current practice, we use the year range instead.

Reply:
Ack. Will change the copyright. Thanks @dhaura and @pavinduLakshan

Reply:
https://github.com/wso2/identity-api-server/pull/634/commits/f6901119a7b6990285779cef13f26dd0fd1df87c